packages/api-client/src/views/Settings/SettingsGeneral.vue

```diff
@@ -158,7 +176,11 @@ const changeTheme = (themeId: ThemeId) => {
                 )
               ">
               <div
-                class="flex items-center justify-center w-5 h-5 rounded-full border-[1.5px] p-1">
+                class="flex items-center justify-center w-5 h-5 rounded-full border-[1.5px] p-1"
+                :class="{
+                  'bg-c-accent border-transparent text-b-1':
+                    !activeWorkspace?.proxyUrl,
+                }">
```

Comment:
```suggestion
                :class="!activeWorkspace?.proxyUrl && 'bg-c-accent border-transparent text-b-1'">
```
I believe you can also use the short circuit for vue class. But whatever you think is better, no preference here

Reply:
i usually do i guess i left as it was, let's do it!